### PR TITLE
doc: add meeting minutes for 2017-10-25

### PR DESCRIPTION
**Pull request by gibfahn**
Fixes: https://github.com/nodejs/build/issues/939

@ev1stensberg feel free to review!

### CODE REVIEW

File: doc/meetings/build-wg-meeting-2017-10-25.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+- Gibson: +1 on pipelines for new projects.
+
+### tmp dir needed on ubuntu 1604 and fedora23 [#873](https://github.com/nodejs/build/issues/873)
+
+- Gibson: I think this is symptomatic of a larger issue, how many people are
+  actually confident in running automation?
+- Refael: Part of the problem is that you need the Jenkins secret to run the
+  Ansible script
+- Rod: For me the biggest problem is how to prioritise the work queue, and I
+  think the Github project is
```
Comment (richardlau) [inline doc/meetings/build-wg-meeting-2017-10-25.md]: is?

**Comment by gibfahn** [comment]
>Maybe remove some of the empty links like Hangouts on Air, for participants

Good point, I missed that. Done.

**Comment by maclover7** [comment]
I don't think it was explicitly an agenda item, but improving interaction process with non-build WG members was discussed. Should a separate issue be opened to discuss this further?

**Comment by gibfahn** [comment]
>I don't think it was explicitly an agenda item, but improving interaction process with non-build WG members was discussed. Should a separate issue be opened to discuss this further?

Possibly, depends exactly who you mean by non-build WG members. https://github.com/nodejs/build/issues/941 is aimed at helping to make it easier for people to get involved.

Happy to open an issue, but not sure what to say in it.

File: doc/meetings/build-wg-meeting-2017-10-25.md [hunk 2]
```diff
@@ -0,0 +1,158 @@
+# Node.js Foundation Build WG Meeting 2017-09-12
```
Comment (jbergstroem) [inline doc/meetings/build-wg-meeting-2017-10-25.md]: Date should be updated or removed

**Comment by refack** [comment]
> Happy to open an issue, but not sure what to say in it.

I think a very short desc of what the Build WG is and does, and how the CI is set up.
* permission groups
* job scripts stuck in Jenkins
* `/ansible/` is future
* `/setup/` is past
* bot / email / website / CI
* CI test vs release

**Comment by gibfahn** [comment]
Opened https://github.com/nodejs/build/issues/950

@maclover7 feel free to comment there, or otherwise tell us what you feel is unclear and we'll fix it.

**Pull request merged**